### PR TITLE
Add VIC Appliance support guide

### PR DESCRIPTION
**Pull request by andrewtchin**
Anything else I should add?

### CODE REVIEW

File: installer/SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+- Is this a fresh deploy of the VIC appliance or an upgrade?
+    - If upgrading, what version is the old VIC appliance?
+- Were any changes made to the disk configuration of the VIC appliance?
+    - If yes, what changes?
+- Are you able to SSH into the VIC appliance? 
+    - If the version is 1.3.1 or greater and able to SSH, please run `/etc/vmware/support/get_vic_appliance_logs.sh` and provide the resulting file to support
+    - If the version is 1.3.0 or less and able to SSH
+        - Run `journalctl -u fileserver`and provide the entire resulting output to support
+        - Run `systemctl status ___`
```
Comment (anchal-agrawal) [inline installer/SUPPORT.md]: Is the `___` intentional?

**Comment by dbarkelew** [comment]
Could this include the configuration scripts that run on the first boot and the configuration scripts that run on reboot?
A section on the locations for the certificate and config files for admiral, harbor and the fileserver service.

**Comment by dbarkelew** [comment]
@anchal-agrawal The documents does have a cert reference. Anything missing from the reference? You were meaning a support checklist to follow, a link to this reference might work for the ova deployment.
https://vmware.github.io/vic-product/assets/files/html/1.3/vic_vsphere_admin/vic_cert_reference.html

**Comment by andrewtchin** [comment]
@anchal-agrawal @dbarkelew Good call I have a change in progress that simplifies certs but I will update to include info for older versions

File: installer/SUPPORT.md (hunk 1 above)
Comment (andrewtchin) [reply installer/SUPPORT.md]: thanks fixed

**Comment by andrewtchin** [comment]
Added some cert info and opened #1334 for more detail

**Pull request merged**